datafusion/sqllogictest/test_files/subquery.slt

```diff
@@ -531,13 +531,13 @@ query TT
 explain SELECT t0_id, t0_name FROM t0 WHERE EXISTS (SELECT 1 FROM t1 INNER JOIN t2 ON(t1.t1_id = t2.t2_id and t1.t1_name = t0.t0_name))
 ----
 logical_plan
-Filter: EXISTS (<subquery>)
---Subquery:
-----Projection: Int64(1)
-------Inner Join:  Filter: t1.t1_id = t2.t2_id AND t1.t1_name = outer_ref(t0.t0_name)
---------TableScan: t1
---------TableScan: t2
+LeftSemi Join: t0.t0_name = __correlated_sq_2.t1_name
```

Comment:
this actually seems like an improvement to me -- as the plans are now properly lowered